src/liblog/lib.rs

```diff
@@ -301,11 +303,10 @@ pub fn log(level: u32, loc: &'static LogLocation, args: fmt::Arguments) {
     // Completely remove the local logger from TLS in case anyone attempts to
     // frob the slot while we're doing the logging. This will destroy any logger
     // set during logging.
-    let mut logger: Box<Logger + Send> = LOCAL_LOGGER.with(|s| {
-        s.borrow_mut().take()
-    }).unwrap_or_else(|| {
-        box DefaultLogger { handle: io::stderr() }
-    });
+    let mut logger: Box<Logger + Send> = LOCAL_LOGGER.with(|s| s.borrow_mut().take())
+                                                     .unwrap_or_else(|| {
+                                                         box DefaultLogger { handle: io::stderr() }
+                                                     });
```

Comment:
Hm this is a bit of an interesting pattern, @nrc I at least find my self doing something like this pretty frequently (e.g. [here](https://github.com/rust-lang/cargo/blob/8964ea787f588650c2fef6295808ceff50d0c6dc/src/cargo/ops/cargo_rustc/context.rs#L300-L346) and [here](https://github.com/alexcrichton/gcc-rs/blob/master/src/windows_registry.rs#L79-L112), although I may just be weird!

I personally find the previous formatting more readable than the latter at least